### DISCUSSION TITLE
Simple (Modal) Resources In Relation Manager

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#3577] [General]
I'm using the Spatie Activity Log package to keep track of changes to my models. I created a Simple (Modal) Resource for the Activity model, which works great. I'm only using the View functionality, if it matters.

I'm now trying to enable the same view modal within an `ActivitiesRelationManager` for a different model resource. While I can get the modal to appear just fine, it's empty.

I understand that the Simple (Modal) Resource _itself_ cannot have relation managers, but that isn't what I'm trying to do.

For reference, here is the relevant code in `Filament/Resources/RegistrantResource/RelationManagers/ActivitiesRelationManager.php`:

```php
public static function table(Table $table): Table
{
    return ActivityResource::table($table);
}
```

**Comment by danharrin** [comment]
What does your ViewAction look like?

**Reply by intrepidws** [reply]
I'm using the default one.

**Reply by danharrin** [reply]
Do you have a form schema declared for your relation manager, so the modal has something to show?

**Reply by intrepidws** [reply]
I'm an idiot. Thanks for pointing me in the right direction.
